src/OpenTelemetry/README.md

```diff
@@ -111,6 +75,101 @@ using var openTelemetry = TracerProviderSdk.EnableTracerProvider(builder => buil
 
 This should link to the Instrumentation documentation.
 
+## Advanced topics
+
+### Build your own Exporter
+
+#### Trace Exporter
+
+* Exporters should subclass `ActivityExporter` and implement `ExportAsync` and

```

Comment:
C# developer may not understand, what is meant by subclass. We could use something like inherit.

Reply:
Roger that.